.forge-snapshots/CreateBoostWithProtocolFee.snap

```diff
@@ -1 +1 @@
-136754
\ No newline at end of file
+185593
```

Comment:
create boost with fees has increased massively in cost. not sure why this is the case

Reply:
https://github.com/snapshot-labs/boost/pull/33#discussion_r1471360433